crates/turborepo-lib/src/run/task_access.rs

```diff
@@ -245,6 +245,25 @@ impl TaskAccess {
         }
     }
 
+    /// Attempt to load a task traced turbo.json
+    pub fn load_turbo_json(&self, root_turbo_json_path: &AbsoluteSystemPath) -> Option<TurboJson> {
+        if !self.enabled {
+            return None;
+        }
+        let trace_json_path = self.repo_root.join_components(&TASK_ACCESS_CONFIG_PATH);
```

Comment:
In the original code, we first resolve the `dir` arg then join to `TASK_ACCESS_CONFIG_PATH`. Do we no longer need the `dir` part?

Reply:
Not required in this PR we can fix this when we need it but this should be aware of the cache dir as well beacuse the intention is that the config gets cached as an artifact.